### DISCUSSION TITLE
Change default note type?

### DISCUSSION THREAD

**Discussion by rmhish** [TriliumNext/Trilium#3133] [Ideas]
Is it possible to change the default note type?

99% of my notes are type "Plain text" or "Code". But I have to manually adjust each new note I create. It would be nice if I could change the default note type from "Text" to "Plain text". But I cannot find any option for doing this.

**Comment by XXXJetfireXXX** [comment]
Maybe use a note template of the desired type?
https://github.com/zadam/trilium/wiki/Template

**Comment by zadam** [comment]
For code types, note type is inherited from parent note. Meaning, if the parent note is a code note, then new children notes are by default created as code notes as well. Is that not the case for you?

**Reply by rmhish** [reply]
Actually, I think it's working the way you described. I think I just didnt recognize it from the beginning so a lot of my note hierarchy was type "Text". I  need to go through and switch all the sub-root notes to "Plain text" now. :) Thanks for the short and simple explanation.
